pkgs/by-name/ti/tiramisu/package.nix

```
@@ -10,12 +10,11 @@
 stdenv.mkDerivation rec {
   pname = "tiramisu";
   # FIXME: once a newer release in upstream is available
-  version = "2.0-unstable-2023-03-29";
+  version = "2.0.20240610";
 
   src = fetchFromGitHub {
     owner = "Sweets";
     repo = "tiramisu";
-    # FIXME: use the current HEAD commit as upstream has no releases since 2021
     rev = "5dddd83abd695bfa15640047a97a08ff0a8d9f9b";
```

Might as well do this:

```suggestion
    tag = version;
```